support-workers/src/main/scala/com/gu/support/workers/PreviewPaymentSchedule.scala

```diff
@@ -12,22 +12,16 @@ object PreviewPaymentSchedule {
 
   def preview(
       subscribeItem: SubscribeItem,
-      billingPeriod: BillingPeriod,
       zuoraService: ZuoraSubscribeService,
       singleResponseCheck: Future[List[PreviewSubscribeResponse]] => Future[PreviewSubscribeResponse],
   ): Future[PaymentSchedule] = {
-    val numberOfInvoicesToPreview: Int = billingPeriod match {
```

Comment:
This was wrong, the number is not the number of invoices, it's the number of months to invoice within